src/status_im2/contexts/chat/composer/utils.cljs

```diff
@@ -114,7 +114,7 @@
 
 (defn calc-shell-neg-y
   [insets maximized? extra-height]
-  (let [padding 6
+  (let [padding 12
```

Comment:
Why this change?

Reply:
Previously the `floating-shell-button` had a padding of `5px` which I removed in this PR. To compensate for this modification, I have adjusted the padding here.